projects/RabbitMQ.Client/client/impl/RabbitMQActivitySource.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading;
+using RabbitMQ.Client.Events;
+using RabbitMQ.Client.Framing.Impl;
+
+namespace RabbitMQ.Client
+{
+    public static class RabbitMQActivitySource
+    {
+        internal const string MessageId = "messaging.message.id";
```

Comment:
it'll be good to point to the semantic conventions doc from which these attributes names are coming from.

Reply:
Yes. I'll add that as a comment.

Reply:
Fixed